parsec/core/remote_devices_manager.py

```diff
@@ -19,12 +16,10 @@
     BackendNotAvailable,
     BackendCmdsNotFound,
 )
-from parsec.core.types import (
-    UnverifiedRemoteDevice,
-    UnverifiedRemoteUser,
-    VerifiedRemoteDevice,
-    VerifiedRemoteUser,
-)
+from parsec.core.trustchain import TrustchainContext, TrustchainError
+
+
+DEFAULT_CACHE_VALIDITY = 60 * 60  # 1h
```

Comment:
Can we justify this value? Why not 5 minutes or 5 hours?

Reply:
No scientific reason here